src/main/java/org/opensrp/domain/postgres/PgStockAndProductDetails.java

```diff
@@ -1,4 +1,4 @@
-package org.opensrp.domain;
+package org.opensrp.domain.postgres;
```

Comment:
Why is this moved. This class shoould not be updated so that it continues to work for implementations that don't need to access the product catalogou